tests/utils/test_env.py

```diff
@@ -159,7 +159,7 @@ def test_activate_activates_non_existing_virtualenv_no_envs_file(
     m.assert_called_with(
         Path(tmp_dir) / "{}-py3.7".format(venv_name),
         executable="python3.7",
-        flags={"always-copy": False},
+        flags={"always-copy": False, "system-site-packages": False},
```

Comment:
Would be great to move this to a fixture so future additions or changes to defaults would be cleaner.